test/runtests.jl

```diff
@@ -3,11 +3,12 @@
 using Test
 
 using LegendGeSim
+using MJDSigGen
 using LegendHDF5IO, HDF5
 using LegendTestData
```

Comment:
And please resolve the merge conflicts in `test/runtests.jl`
You probably can fix them by removing `HDF5` from line 7